### DISCUSSION TITLE
Setting up a private repo to enforce branch protections

### DISCUSSION THREAD

**Discussion by rfalanga** [community/community#23597] [New to GitHub]
<p>I’m setting up a branch protection on my main branch in one of my private GitHub repos. Currently I’m the only one working with this code, however I want to check the “Include administrators” option. And I’ve also checked the “Require a pull request before merging” option with one approval being required. I assume that means I can approve my own PRs. Am I correct about that?</p>
<p>I recently upgraded my GitHub account to GitHub Pro.</p>

**Comment by Katsute** [comment, marked as answer]
<p>“Include administrators” means that the branch protection rules apply to admins, with no override.</p>
<p>GitHub does not currently allow self approvals.</p>
